app/controllers/api.php

```diff
@@ -53,6 +53,9 @@
     case 'versions':
         $json = include MODELS . 'api/versions.php';
         break;
+    case 'transliterate':
+        include MODELS . 'api/transliterate.php';
```

Comment:
You should do like other cases and create the $json variable there and not in the model:
```php
$json = include MODELS . 'api/transliterate.php';
```